pyproject.toml

```diff
@@ -10,7 +10,7 @@ repository = "https://github.com/ollama/ollama-python"
 
 [tool.poetry.dependencies]
 python = "^3.8"
-httpx = "^0.27.0"
+httpx = ">=0.27.0,<0.28.0"
```

Comment:
this has the same semantics so it's unnecessary https://python-poetry.org/docs/dependency-specification/#caret-requirements